pkg/template/crud/create.go

```diff
@@ -0,0 +1,36 @@
+// Create file for {{.Singular}} instances
```

Comment:
one optional change. We can have this as one template, instead of create, delete, describe, list, use as different templates. As just names are changed and rest all is repeated. I think this can be done by making a map of names and a loop. @namit-chandwani @wtrocki 

Overall works as expected 🤞

Reply:
Good catch!
Initially, I had assumed that all CRUD files would contain something unique, which is why I had started with separate files.
Since that is not the case anymore, I'll change it now :)

Reply:
-1000000 please not do it. This is optimalization based on current state. We will add specific code to it later

Reply:
Actually we need common flags without them commands will not work